### DISCUSSION TITLE
Allow editing relations on view page

### DISCUSSION THREAD

**Discussion by Rosendito** [filamentphp/filament#12432] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.70

### How can we help you?

Do anyone know how to allow to relation editing (using relation manager) on view page?

It seems weird to me have to move to edit page to create child relations in my resource. I think it's a bad user experience.

**Comment by dissto** [comment, marked as answer]
Do you mean [Read-only mode](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#read-only-mode)?

**Reply by Rosendito** [reply]
Ty so much sir.

**Reply by alsahlawi** [reply]
Thanks, that was helpful

**Reply by khdfierror** [reply]
tysm sir

**Reply by aziz-prasetyo** [reply]
Thank you.
